src/lib/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, ComponentProps, PropsWithChildren } from 'react';
+import { HiOutlineChevronRight } from 'react-icons/hi';
+
+export interface BreadcrumbItemProps extends ComponentProps<'li'> {
+  href?: string;
+  icon?: FC<ComponentProps<'svg'>>;
+  isFirst?: boolean;
+}
+
+const BreadcrumbItem: FC<PropsWithChildren<BreadcrumbItemProps>> = ({
+  children,
+  href,
+  icon: Icon,
+  isFirst,
+}): JSX.Element => {
+  return (
+    <li className="flex items-center">
+      {!isFirst && (
```

Comment:
I'm not sure if what I'm suggesting gonna work: add a `group` class to the parent `li` element and use `group-first:hidden` class instead of this isFirst prop

Reply:
This works great!